src/os/time/time.js

```diff
@@ -882,3 +882,22 @@ os.time.step = function(date, duration, offset, opt_local) {
   var next = os.time.offset(date, duration, offset, opt_local);
   return (next.getTime() - date.getTime());
 };
+
+
+/**
+ * Combines the days and time for seperate mapped fields
+ * @param {string} days The year, month, day
+ * @param {string} time The hours, minutes, seconds, milliseconds
+ * @param {string} daysFormat The format for days
```

Comment:
It might be useful to assume ISO 8601 format, so the `daysFormat` and `timeFormat` fields can be optional.